examples/lossy.py

```diff
@@ -0,0 +1,52 @@
+##############################################################################
+# blosc2_grok: Grok (JPEG2000 codec) plugin for Blosc2
+#
+# Copyright (c) 2023  The Blosc Development Team <blosc@blosc.org>
+# https://blosc.org
+# License: GNU Affero General Public License v3.0 (see LICENSE.txt)
+##############################################################################
+
+import blosc2
+import argparse
+import numpy as np
+from PIL import Image
+import os
+import blosc2_btune
+
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Compress the given input image using Blosc2 and grok',
+    )
+    add_argument = parser.add_argument
+    add_argument('inputfile')
+    add_argument('-o', '--outputfile', type=str, help='File name from decompressed image')
+    args = parser.parse_args()
+    im = Image.open(args.inputfile)
+
+    # Set Btune parameters
+    base_dir = os.path.dirname(__file__)
+    kwargs = {
+        "tradeoff": (0.5, 0.3, 0.2),
```

Comment:
Please add a comment with the meaning of the tradeoff (triplet version)

Reply:
Fixed in 9fcfb27cc73198307be5bb26ef401e393db473a5.